classes/class-ledyer-main.php

```diff
@@ -175,9 +175,9 @@ public function process_notification( $ledyer_order_id ) {
 				\Ledyer\Logger::log( 'Couldn\'t acknowledge order ' . $ledyer_order_id  );
 				return;
 			}
-			$ledyer_update_order = ledyer()->api->update_order_reference( $ledyer_order_id, array( 'reference' => strval( $order->ID ) ) );
+			$ledyer_update_order = ledyer()->api->update_order_reference( $ledyer_order_id, array( 'reference' => $order->get_order_number() ) );
```

Comment:
Do other plugins out there do this as well?

It seems strange that Maksimer would do this if it was a non-standard way

Reply:
yes. get_order_number fallbacks to id if no custom format is used

Reply:
great!